identity/app/actions/AuthenticatedActions.scala

```diff
@@ -38,11 +38,11 @@ class AuthenticatedActions(
     SeeOther(identityUrlBuilder.buildUrl(redirectUrlWithParams))
   }
 
-  def sendUserToConsentJourney(request: RequestHeader): Result =
-    redirectWithReturn(request, "/consent?journey=repermission")
+  def sendUserToAllConsentsJourney(request: RequestHeader): Result =
+    redirectWithReturn(request, "/consents/all")
```

Comment:
can you replace `/consents/all` with `routes.EditProfileController.displayConsentJourney.url` (or similar)?

Reply:
Separate PR.